package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "couchbase": "github:couchbase/couchnode#master"
```

Comment:
Should this not point to the latest 3.1.X version(tag) instead of master? Or else I think we'll be pulling the latest dev version of the SDK which seems to be [3.2.0](https://github.com/couchbase/couchnode/blob/master/package.json#L58).

Reply:
OK, so the code sample doesn't work with 3.1.3, I assume it'll be backported eventually, but may be simpler to keep master for now...?

Would be good to have our example automation verify which versions work correctly for different snippets perhaps?